client/package.json

```diff
@@ -12,7 +12,7 @@
     "tw": "rm -rf .elm-tailwind-modules && node scripts/generateTailwind.js",
     "setup": "npm ci && npx elm-tooling install && npm run tw",
     "build:docker": "npm run setup && npm run config:prod && npm run api:local && npm run build",
-    "build:github": "npm run setup && npm run config:local && npm run api:schema && npm run api:local && npm run build",
+    "build:github": "npm run setup && npm run config:local && npm run api:schema && elm-spa gen && npm run api:local && vite build",
```

Comment:
elm-graphql's validation step needs a valid elm.json, which effectively needs elm-spa generation to have been run